Dockerfile

```diff
@@ -0,0 +1,5 @@
+FROM mhart/alpine-node
+
+RUN npm install -g hercule@latest
```

Comment:
Above I've suggested the `mhart/alpine-node:base` which excludes NPM. This image is smaller and skips the `npm install` step, but we'll need to run the install process before building the image.